src/environmentd/tests/sql.rs

```diff
@@ -1343,3 +1407,92 @@ fn get_explain_timestamp(table: &str, client: &mut postgres::Client) -> EpochMil
     let timestamp_caps = timestamp_re.captures(&explain).unwrap();
     timestamp_caps.get(1).unwrap().as_str().parse().unwrap()
 }
+
+/// Helper function to create a Postgres source.
+///
+/// IMPORTANT: Make sure to call `cleanup_postgres_source` at the end of the test.
```

Comment:
You can have this function return a closure to a cleanup fn that users call. A bit cleaner because it avoids having to correctly call again with the same arguments.

Reply:
Good call, just updated.